pages/heroku.md

```diff
@@ -131,6 +131,115 @@ Or when using gradle:
 
 `heroku deploy:jar build/libs/*jar`
 
+## Deploying Docker to Heroku
+
+You can deploy your app as a Docker container to Heroku too. While this works, there's no Heroku setup and configuration that happens, so you have to do that manually. This documentation assumes you've already run `jhipster heroku` to deploy your app and therefore leverages the integration and add-on provisioning that this process performs.
+
+Build your Docker image:
```

Comment:
@atomfrede Do you think I should include [these changes](https://github.com/jhipster/generator-jhipster/pull/12237) as a note for developers using JHipster < 6.10.2?

Reply:
Yes, guess that would prevent stackoverflow questions or issues on our side.